mesh_segmenter/src/mesh_segmenter.cpp

```diff
@@ -48,9 +34,9 @@ std::vector<vtkSmartPointer<vtkPolyData> > MeshSegmenter::getMeshSegments()
   vtkSmartPointer<vtkPolyData> input_copy = vtkSmartPointer<vtkPolyData>::New();
   input_copy->DeepCopy(input_mesh_);
   std::vector<vtkSmartPointer<vtkPolyData> > meshes;
-  for (int i = 0; i < included_indices_.size(); ++i)
+  for (std::size_t i = 0; i < included_indices_.size(); ++i)
   {
-    LOG4CXX_INFO(SEGMENTATION_LOGGER, "Segment " << i << " size: " << included_indices_.at(i)->GetNumberOfIds());
+    CONSOLE_BRIDGE_logInform(("Segment " + std::to_string(i) + " size: " + std::to_string(included_indices_.at(i)->GetNumberOfIds())).c_str());
```

Comment:
Why console bridge over log4cxx?

Reply:
We were getting log4cxx from rosconsole. Switching to console_bridge seemed like the thing to do to make this ROS agnostic.

Reply:
Also I believe when using the ROS agnostic package within the ros ecosystem you can pipe it to ros log files.